puzzles/puzzle1.html

```diff
@@ -30,113 +30,145 @@
-    <li>In the cellPanel div, add your own div and give it a class of 'camperCell'</li>
-    <li>Give your div a unique id</li>
-    <li>Inside your div, add an h1 element that has your name</li>
-    <li>Inside your div, add an image that has a picture/avatar of you</li>
-    <li>Inside your div, add a p element that contains your favorite quote</li>
-    <li>Add any custom styling by targeting your custom id in the provided style tag</li>
-  </ol>
-</div>
+    <p>
+      <u><strong>Description:</strong></u> Now that you've added your name to
+      the contributors.md file, its time to change some actual code. The
+      following exercise will have you add your own camper cell, html elements,
+      and custom styling.
+    </p>
+    <p>
+      <u><strong>Puzzle:</strong></u>
+    </p>
+    <ol>
+      <li>
+        In the cellPanel div, add your own div and give it a class of
+        'camperCell'
+      </li>
+      <li>Give your div a unique id</li>
+      <li>Inside your div, add an h1 element that has your name</li>
+      <li>Inside your div, add an image that has a picture/avatar of you</li>
+      <li>
+        Inside your div, add a p element that contains your favorite quote
+      </li>
+      <li>
+        Add any custom styling by targeting your custom id in the provided style
+        tag
+      </li>
+    </ol>
+  </div>
 
-<div id='yourWork'>
+  <div id="yourWork">
+    <h1>Camper Cells</h1>
 
-  <h1>Camper Cells</h1>
+    <div id="cellPanel">
+      <!-- Add your camperCell Here -->
 
-  <div id='cellPanel'>
+      <div class="camperCell" id="davidfraser">
+        <h1>David Fraser</h1>
+        <img
+          src="https://avatars2.githubusercontent.com/u/47202564?s=460&u=35c8f87e6b9e00267c22664efd668db3b4ad7491&v=4"
+          alt="davidProfPic"
+        />
+        <p>
+          "Alone we can do so little; together we can do so much." – Helen
+          Keller
+        </p>
+      </div>
 
-  <!-- Add your camperCell Here -->
+      <div class="camperCell" id="camper2">
+        <h1>Camper2</h1>
+        <img
+          src="https://avatars1.githubusercontent.com/u/583231?s=460&u=a59fef2a493e2b67dd13754231daf220c82ba84d&v=4"
+          alt="davidProfPic"
+        />
+        <p>
+          “Be the change that you wish to see in the world.” ― Mahatma Gandhi
+        </p>
+      </div>
 
-  <div class='camperCell' id='davidfraser'>
-    <h1>David Fraser</h1>
-    <img src='https://avatars2.githubusercontent.com/u/47202564?s=460&u=35c8f87e6b9e00267c22664efd668db3b4ad7491&v=4' alt='davidProfPic'>
-    <p>"Alone we can do so little; together we can do so much." – Helen Keller</p>
-  </div>
-
-  <div class='camperCell' id='camper2'>
-    <h1>Camper2</h1>
-    <img src='https://avatars1.githubusercontent.com/u/583231?s=460&u=a59fef2a493e2b67dd13754231daf220c82ba84d&v=4' alt='davidProfPic'>
-    <p>“Be the change that you wish to see in the world.” ― Mahatma Gandhi</p>
-  </div>
+      <div class="camperCell" id="camper2">
```

Comment:
Nice! Thanks for doing the puzzle! 
One thing: looks like you're using the same 'id' as camper 2. Try adding your own id and custom style - which will be a walk in the park for you 😄.

Reply:
sorted that now mate 😎👍